snapfaas/src/sched/messages.proto

```diff
@@ -0,0 +1,61 @@
+syntax = "proto3";
+
+package snapfaas.sched.messages;
+
+// Begin of request messages
+message GetJob {
+    uint64 id = 1;
+}
+
+message FinishJob {
+    string id     = 1;
```

Comment:
why is this a `string` while the `id` in `GetJob` is a `unit64`?

Reply:
`id` in `GetJob` is the thread id and `id` in `ProcessJob` and `FinishJob` are the task id. `id` in `GetJob` is originally reserved for identifying the worker itself in a persistent connection. But yeah, the name is confusing, and I'll change it.